docs/reference/datatiers.asciidoc

```diff
@@ -37,9 +37,9 @@ TIP: The performance of an {es} node is often limited by the performance of the
 For example hardware profiles, refer to Elastic Cloud's {cloud}/ec-reference-hardware.html[instance configurations]. 
 Review our recommendations for optimizing your storage for <<indexing-use-faster-hardware,indexing>> and <<search-use-faster-hardware,search>>.
 
-IMPORTANT: {es} generally expects nodes within a data tier to share the same 
-hardware profile. Variations not following this recommendation should be 
-carefully architected to avoid <<hotspotting,hot spotting>>.
+IMPORTANT: {es} assumes nodes within a data tier share the same hardware profile (such as CPU, RAM, disk capacity) 
+as it spreads the load equally across all nodes within the data tier.
+Variations not following this recommendation should be carefully architected to avoid <<hotspotting,hot spotting>>.
```

Comment:
consider

```suggestion
Data tiers with unequally resourced nodes are at risk of <<hotspotting,hot spotting>>.
```

Reply:
I agree with your point. 

I believe the phrasing is intended to push the customer towards, while you may get it to work, Elastic doesn't "support" figuring this out for you. If we leave it at "nodes are at risk", we're going to keep getting supportCases+devEscalations which I believe is why we're updating this to say "you did it to yourself" by going outside Elasticsearch assumptions. Could we workshop a middle ground phrasing?

Reply:
how about

```suggestion
Data tiers with unequally resourced nodes are at risk of <<hotspotting,hot spotting>>. These configurations are not supported.
```

I think the current phrasing will lead to people looking for guidance of what "careful" could look like